controller.java

```diff
@@ -6,7 +6,7 @@ public static void main(String[] args) {
         int num2 = 20;
 
         // Perform addition
-        int sum = num1 + num2;
+        int sum = math.add(num1, num2);
```

Comment:
you can consider this:
```suggestion
        int summation = math.add(num1, num2);
```